modules/frontend/src/components/pages/DataQualityPage.vue

```diff
@@ -5,14 +5,47 @@
       <div class="container-fluid">
         <div class="ms-auto d-flex align-items-center text-white">
           <i class="bi bi-person-fill fs-4 me-2"></i>
-          <span>{{ username }}</span>
+          <!-- Warning indicator for default password -->
+          <div v-if="defaultPasswordFlag" class="position-relative me-2"
+               @mouseenter="showTooltip = true"
+               @mouseleave="showTooltip = false">
+            <i class="bi bi-exclamation-triangle-fill text-warning fs-5"></i>
+            <div v-if="showTooltip" class="custom-tooltip">
+              You are using the default password. Please change it for security.
```

Comment:
We should make this text more formal/professional

Reply:
eg. For your account's security, please change your password.
Your current password is a system-generated default.